doc/integration_docs/4_builtinpackages.md

```diff
@@ -7,22 +7,25 @@ The Built-In Packages mechanism is an effort to bundle more node content with Dy
+* Semantic versioning, it's probably a good idea to version your package using a semantic versioning scheme, and to communicate it to users in the package description or docs.
 * Automated tests! Please see above, if a package is included using the Built-In Package mechanism, to a user it appears to be part of the product and should be tested like a product.
 * High level of polish: icons, node docs, localized content.
-* ... others TBD
+* Don't ship packages that you or your team cannot maintain.
+* Don't ship third party packages this way (see above).
 
+Basically, you should have full control over the package, the ability to fix it, keep it updated, and to test it against the latest changes in Dynamo and your product. You also need the ability to sign it.
 
-### Built-In Packages versus Host Integration Specific Packages
 
-We are intending the `Built-In Packages` to be a core feature, a set of packages that all users gain access to, even if they do not have access to the package manager. The underlying mechanism to support this feature is an additional default loading location for packages directly in the dynamo core directory - relative to DynamoCore.dll.
+### Built-In Packages versus Host Integration Specific Packages
 
-With some constraints this location will be useable for ADSK Dynamo clients and integrators to ship integration specific packages.
+We are intending the `Built-In Packages` to be a core feature, a set of packages that all users gain access to, even if they do not have access to the package manager. Currently the underlying mechanism to support this feature is an additional default loading location for packages directly in the dynamo core directory - relative to DynamoCore.dll.
 
-Because the underlying loading mechanism is the same - it will be necessary to make sure that packages included this way do not lead to user confusion about core `Built-In Packages` packages, and integration specific packages that are only available in a single host product. We advise that until the Dynamo team designs and implements apis to support this - that to avoid user confusion - the `Built-In Packages` should only be used in discussion with the team.
+With some constraints this location will be useable for ADSK Dynamo clients and integrators to distribute their integration specific packages. *(for example, the Dynamo Formit integration requires a custom Dynamo Formit package).*
 
+Because the underlying loading mechanism is the same for both core and host specific packages - it will be necessary to make sure that packages distributed this way do not lead to user confusion about core `Built-In Packages` packages vs. integration specific packages that are only available in a single host product. We advise that that to avoid user confusion - the `Built-In Packages` should be used in discussion with the Dynamo teams.
```

Comment:
I guess this section was a bit unclear to me.
Not sure exactly where the user confusion is. Is it because there is no way to differentiate between them at the NodeLibrary level ?

Reply:
yeah, pretty much, Let's have an example like a Revit only package that is distributed with Revit that includes some new list nodes, the authors decide to move the nodes in the package under the main `List` category.

User's who encounter these nodes will think they are from core, but they won't exist when they try to open the same graph in sandbox.

Instead of the text above about working with the Dynamo teams I could just provide an example like that - what do you think?

Reply:
How about this: `We advise that that to avoid user confusion - the host-specific packages should be introduced in discussion with the Dynamo teams.`